src/briefcase/platforms/macOS/app.py

```diff
@@ -61,9 +67,15 @@ def install_app_support_package(self, app: AppConfig):
                 self.tools.shutil.rmtree(runtime_support_path)
             runtime_support_path.mkdir()
 
+            # The support package will contain a lot more content than is
+            # actually needed at runtime. Only copy the runtime_path into
+            # the support location nominated by the template. Also ensure
+            # that symlinks are preserved.
+            runtime_path = self.runtime_path(app)
             self.tools.shutil.copytree(
-                self.support_path(app) / "python-stdlib",
-                runtime_support_path / "python-stdlib",
+                self.support_path(app) / runtime_path,
+                runtime_support_path / Path(runtime_path).name,
+                # symlinks=True,
```

Comment:
Isn't this supposed to be uncommented? If so, the tests should also be improved so they would have caught this.

Reply:
Dangit... I commented this out specifically to verify that the test was working... and then the test passed, so I didn't add the extra test that was required.